starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,71 @@
+require ('../app');
+const express = require('express');
+const router  = express.Router();
+
+const Celebrity = require ("../models/celebrity") //los models siempre con mayuscula 
+
+
+router.get('/celebrities', (req, res) => {  //el primer fallo de crear el archivo celebrities.js fue pasar este codigo alli, moviendo este codigo de nuevo a index.js ya funciona, pendiente revisar si esto es correcto
+    Celebrity.find()
+        .then((theCelebrity) => {
+            console.log(theCelebrity)
+            res.render('celebrities/index.hbs', {theCelebrity})
+        })
+        .catch(err => console.log('Error', err))
+    
+        })
+  
+      
+router.get('/celebrities/:id', (req, res, next) => {
+    Celebrity.findById(req.params.id)
+        .then(celebrity => res.render('celebrities/show.hbs', {celebrity}))
+        .catch(error => console.log("Oppsss, an error happened", error))
+          
+      })
+
+    //create new celebrity
+
+router.get('/celebrities/news', (req, res) =>{
+      res.render('celebrities/news.hbs')
+    })
+
+router.post('/celebrities/news', (req, res, next) => {
+      let {name, occupation, catchPhrase} = req.body
+    console.log(req.body)
+    Celebrity.create(req.body)
+          .then(() => {
+            res.render('../views/celebrities/news.hbs', {successCelebrity: true})
+          })
+            .catch(error => console.log("Oppsss, an error happened", error))
```

Comment:
you always want to handle errors differently. Right now the server will know what error is happening via the console.log, however the user will just be stuck in a loading without anything to render. There are several approaches to work around this. You can put a next(), meaning the route will stop there and the server will move to the next routes, eventually throwing the 404 error page. Or you can render the same page again but with an error message.